hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -715,14 +717,52 @@ ppResult name (Report tests discards result) =
         ppDiscardCount discards <>
         ", passed" <+>
         ppTestCount tests <>
-        "."
+        "." <+>
+        ppClassification classes <+>
+        ppCoverage classes
 
     OK ->
       pure . icon SuccessIcon '✓' . WL.annotate SuccessHeader $
         ppName name <+>
         "passed" <+>
         ppTestCount tests <>
-        "."
+        "." <+>
+        ppClassification classes <+>
+        ppCoverage classes
+
+ppClassification :: Classification -> Doc Markup
+ppClassification (Classification cls total)
+  | Map.null cls = mempty
+  | otherwise = (<>) WL.linebreak $ WL.indent 4 . WL.align . WL.vsep $
+    (\(k, v) -> WL.text $ show (occurrenceRate v total) <> "% " <> k) <$> Map.toList cls
```

Comment:
Sorry I missed this one in the review, can you change these guards to an `if` expression so we can keep two space indenting:

```haskell
  if Map.null cls then
    mempty
  else
    (<>) WL.linebreak $ WL.indent 4 . WL.align . WL.vsep $
      (\(k, v) -> WL.text $ show (occurrenceRate v total) <> "% " <> k) <$> Map.toList cls